### DISCUSSION TITLE
Support for Punjabi

### DISCUSSION THREAD

**Discussion by karanbirsingh99** [explosion/spaCy#12308] [Language Support]
I looked at all the current languages that you support that there aren't currently models for and I see that there is Gujrati and Hindi, which are in the same language family as Punjabi. I would love if there was a NLP for Punjabi, and was wondering what the prerequisites/requirements to support the language and possibly train a model for it would be. I'm really new to this space so any recommended reading would also be quite helpful.

**Comment by svlandeg** [comment]
Hi, welcome to the spaCy space! ;-)

You can read up about support for new languages in Ines' thread here: https://github.com/explosion/spaCy/discussions/3056. It gives an overview of the levels of support we can offer for different languages and what is needed for each step. In general, we heavily rely on contributions from native speakers to help us with these. The post also includes some general pointers to documentation about various spaCy concepts, but otherwise a good place to start reading is here: https://spacy.io/usage/spacy-101

**Reply by karanbirsingh99** [reply]
I will check these resources out. Thank you for the prompt reply!
